planet4-gutenberg-blocks.php

```diff
@@ -138,6 +138,7 @@
 
 const BETA_PAGE_BLOCK_TYPES = [
 	'planet4-blocks/covers-beta',
+	'planet4-blocks/carousel-header-beta',
```

Comment:
Let's only add them when merging the wysi PR? Otherwise this will be an identical version to the previous :smile:

Reply:
Although just adding them here does not allow them in the editor. So we can actually already add it I guess.

Reply:
Removed 🙂